models/draft.js

```diff
@@ -1,31 +1,55 @@
 const mongoose = require('mongoose');
 const cardSchema = require('./cardSchema');
 
+// Details on each pack, how to draft and what's in it.
+const Pack = {
+  cards: [
+    {
+      type: Number,
+      min: 0,
+    },
+  ],
+  pickAtTime: {
```

Comment:
not a fan of this name. Maybe `cardsPerPick` ?

Reply:
how about cardsPerPass?